packages/forms-web-app/src/routes/before-you-start/enforcement-notice.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+
+const fetchExistingAppealMiddleware = require('../../middleware/fetch-existing-appeal');
+const enforcementNoticeController = require('../../controllers/before-you-start/enforcement-notice');
+const { validationErrorHandler } = require('../../validators/validation-error-handler');
+const {
+  rules: enforcementNoticeValidationRules,
+} = require('../../validators/before-you-start/enforcement-notice');
+
+const router = express.Router();
+
+router.get(
+  '/use-a-different-service',
+  enforcementNoticeController.getServiceNotAvailableWhenReceivedEnforcementNotice
+);
+router.get(
+  '/enforcement-notice',
```

Comment:
I don't think we need to collect the existing appeal for the get

Reply:
As discussed, leaving it for the moment